README.md

```diff
@@ -73,6 +75,7 @@ open http://127.0.0.1:8080/
 The finished web app is found in the `docs/` folder (this is so that you can easily share it with [GitHub Pages](https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site)). It consists of three files:
 
 * `index.html`: A few lines of HTML, CSS and JS that loads your app. **You need to edit this** (once) to replace `eframe_template` with the name of your crate!
+  By default, the offline caching is disabled so that development is easier. To enable it simple uncomment the lines in the JavaScript section near the end.
```

Comment:
Near the end of what? Add a link?

Reply:
within index.html near the end. (it is a comment for that file).

But I've changed the approach - this is hopefully more clear.